tests/test_dynunet.py

```diff
@@ -118,6 +121,32 @@ def test_script(self):
         test_script_save(net, test_data)
 
 
+@skip_if_no_cuda
+@skip_if_windows
+@unittest.skipUnless(has_nvfuser, "To use `instance_nvfuser`, `apex.normalization.InstanceNorm3dNVFuser` is needed.")
+class TestDynUNetWithInstanceNorm3dNVFuser(unittest.TestCase):
+    @parameterized.expand([TEST_CASE_DYNUNET_3D[0]])
+    def test_consistency(self, input_param, input_shape, _):
+        for eps in [1e-4, 1e-5]:
+            for momentum in [0.1, 0.01]:
+                for affine in [True, False]:
+                    norm_param = {"eps": eps, "momentum": momentum, "affine": affine}
+                    input_param["norm_name"] = ("instance", norm_param)
+                    input_param_fuser = input_param.copy()
+                    input_param_fuser["norm_name"] = ("instance_nvfuser", norm_param)
+                    net = DynUNet(**input_param).to("cuda")
+                    net_fuser = DynUNet(**input_param_fuser).to("cuda")
+                    net_fuser.load_state_dict(net.state_dict())
+
+                    input_tensor = torch.randn(input_shape).to("cuda")
```

Comment:
Should we use `channel_last` shape for the input data and net?
And I think maybe we should use `cuda:0` instead of `cuda` as @wyli mentioned before.

Thanks.

Reply:
Hi @Nic-Ma and @wyli , for the `cuda:0` thing, it's my first time to see this requirement, but I found that it's common to use:
```
"cuda" if torch.cuda.is_available() else "cpu"
```
in MONAI code base. Could you help to explain more about it?  Thanks!

Reply:
Yes the tests are running in a multigpu environment and the current device might not be cuda:0, explicitly using cuda:0 can make sure it's using the first visible gpu.